emacs/merlin.el

```diff
@@ -1487,6 +1487,7 @@ Empty string defaults to jumping to all these."
 
 (defun merlin--occurrence-text (line-num marker start end source-buf)
   (concat (propertize (format "%7d:" line-num)
+                      'font-lock-face 'shadow
```

Comment:
What is this about?

Reply:
This sets the style of the line number to use the `shadow` style, which makes it darker.

Otherwise it can be a little hard to read `123:foo bar` that `123:` is a line number. With this change, `123:` is using the `shadow` style and it's visually distinct.

This is also consistent with `M-x occur` :)